site/content/contribute/mobile/build-your-own/android.md

```diff
@@ -83,7 +83,7 @@ To make it easier to customize your build, we've defined a few environment varia
 | `INCREMENT_BUILD_<br>NUMBER_MESSAGE` | Set the commit message when changing the app build number. | Bump app build number to | No |
 | `APP_NAME` | The name of the app as it is going to be shown on the device home screen. | Mattermost Beta | Yes |
 | `APP_SCHEME` | The URL naming scheme for the app as used in direct deep links to app content from outside the app. | mattermost | No |
-| `REPLACE_ASSETS` | Override the assets as described in [White Labeling](contribute/mobile/build-your-own/white-label/). <br><br>Valid values are: `true`, `false` | `false` | No |
+| `REPLACE_ASSETS` | Override the assets as described in [White Labeling](/contribute/mobile/build-your-own/white-label/). <br><br>Valid values are: `true`, `false` | `false` | No |
```

Comment:
Links should make use of [`ref`](https://gohugo.io/functions/ref/#readout) to catch issues like this.